lib/src/utils/validators.dart

```diff
@@ -32,7 +34,7 @@ bool isURL(String? str,
     List<String> hostBlacklist = const []}) {
   if (str == null ||
       str.isEmpty ||
-      str.length > 2083 ||
+      str.length > _maxUrlLength ||
```

Comment:
I added this because I actually didn't know that 2083 was the maximum length for an url